package.json

```diff
@@ -11,14 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "async": "1.5.0",
-    "coffee-script": "1.9.3",
-    "colors": "1.1.2",
-    "jquery": "^2.2.0",
-    "rootpath": "^0.1.2",
-    "async": "1.5.0",
```

Comment:
`async` is required [here](https://github.com/indigotech/bot-devices/pull/12/files/9a42a3bcb605cf241ee3f74c2f78dd806e00849d#diff-d058f94b15d7abb97d499186d40867ffR18)